types/parse/index.d.ts

```diff
@@ -907,22 +907,30 @@ declare namespace Parse {
             object: Object;
         }
 
-        interface BeforeFindTriggerRequest extends TriggerRequest {
-            query?: Query
-            count?: boolean
-        }
 
         interface AfterSaveRequest extends TriggerRequest { }
         interface AfterDeleteRequest extends TriggerRequest { }
         interface BeforeDeleteRequest extends TriggerRequest { }
         interface BeforeDeleteResponse extends FunctionResponse { }
         interface BeforeSaveRequest extends TriggerRequest { }
-        interface BeforeFindRequest extends BeforeFindTriggerRequest { }
         interface BeforeSaveResponse extends FunctionResponse {
             success: () => void;
         }
+
+        // Read preference describes how MongoDB driver route read operations to the members of a replica set.
+        enum ReadPreferenceOption {
+            Primary = 'PRIMARY',
```

Comment:
Does this enum actually exist at runtime? Can you provide documentation for the change?